pyaedt/modules/Boundary.py

```diff
@@ -2,6 +2,7 @@
 This module contains these classes: `BoundaryCommon` and `BoundaryObject`.
```

Comment:
```suggestion
This module contains these classes: ``BoundaryCommon`` and ``BoundaryObject``.
```
Double backticks format as code entities. A single backtick formats only in italic type.